docker/README.md

```diff
@@ -0,0 +1,112 @@
+    b8e1de41deb5        fiware-aiakos       "/bin/sh -c ./start.s"   6 minutes ago       Up 6 minutes        0.0.0.0:3000->3000/tcp   fervent_davinci
+
+
+Take the Container ID and execute `docker stop b8e1de41deb5` or `docker kill b8e1de41deb5`. Note that you will lose any data that was being used in Aiakos using this method.
+
+----
+## 2. Run Unit Test of Aiakos
+
+Taking into account that you download the repository from GitHub (See Section **1. The Fastest Way**), this method will launch a container running Aiakos, and execute the unit tests associated to the component. You should move to the UnitTests folder `./UnitTests`. Just create a new docker image executind `docker build -t fiware-aiakos-unittests -f Dockerfile .`. Please keep in mind that if you do not change the name of the image it will automatically create a new one for unit tests and change the previous one to tag none.
+
+To see that the image is created run `docker images` and you see something like this:
+
+    REPOSITORY                TAG                 IMAGE ID            CREATED             SIZE
+    fiware-aiakos-unittests   latest              103464a8ede0        30 seconds ago      551.3 MB
+    centos                    latest              904d6c400333        2 weeks ago         196.8 MB
+
+To execute the unit tests of this component, just execute `docker run --name fiware-aiakos-unittests fiware-aiakos-unittests`. Finally you can extract the information of the executed tests just executing `docker cp fiware-aiakos-unittests:/opt/fiware-aiakos/report .`
+
+
+> TIP: If you are trying these methods or run them more than once and come across an error saying that the container already exists you can delete it with `docker rm fiware-aiakos-unittests`. If you have to stop it first do `docker stop fiware-aiakos-unittests`.
+
+Keep in mind that if you use these commands you get access to the tags and specific versions of Aiakos. If you do not specify a version you are pulling from `latest` by default.
+
+----
+## 3. Run Acceptance tests
+
+Taking into account that you download the repository from GitHub (See Section **1. The Fastest Way**). This method will launch a container to run the E2E tests of the Aiakos component, previously you should launch or configure a FIWARE Lab access. You have to define the following environment variables:
+
+    export BRANCH=develop
+    export KEYSTONE_IP=<IP of the keystone instance>
+    export ADM_TENANT_ID=<admin tenant id in the OpenStack environment>
+    export USER_TENANT_ID=<admin tenant id in the OpenStack environment>
+    export ADM_TENANT_NAME=<admin tenant name>
+    export USER_TENANT_NAME=<user tenant name>
+    export ADM_USERNAME=<admin username>
+    export USER_USERNAME=<username>
+    export ADM_PASSWORD=<admin password>
+    export USER_PASSWORD=<user password>
+    export Region1=qaregion
```

Comment:
I wanted to say, that for aiakos acceptance tests the region should be called qaregion. In case it is not, tests are going to fail